src/DurableTask.Netherite/StorageLayer/Faster/AzureBlobs/BlobManager.cs

```diff
@@ -91,25 +91,26 @@ public class FasterTuningParameters
             public int? NumPagesToPreload;
         }
 
-        public FasterLogSettings GetDefaultEventLogSettings(bool useSeparatePageBlobStorage, FasterTuningParameters tuningParameters) => new FasterLogSettings
+        public FasterLogSettings GetEventLogSettings(bool useSeparatePageBlobStorage, FasterTuningParameters tuningParameters)
         {
-            LogDevice = this.EventLogDevice,
-            LogCommitManager = this.UseLocalFiles
-                ? null // TODO: fix this: new LocalLogCommitManager($"{this.LocalDirectoryPath}\\{this.PartitionFolderName}\\{CommitBlobName}")
-                : (ILogCommitManager)this,
-            PageSizeBits = tuningParameters?.EventLogPageSizeBits ?? 21, // 2MB
-            SegmentSizeBits = tuningParameters?.EventLogSegmentSizeBits ??
-                (useSeparatePageBlobStorage ? 35  // 32 GB
-                                            : 26), // 64 MB
-            MemorySizeBits = tuningParameters?.EventLogMemorySizeBits ?? 22, // 2MB
-        };
+            (int pageSizeBits, int segmentSizeBits) = BlobManager.GetImmutableEventLogParameters(useSeparatePageBlobStorage, tuningParameters);
 
-        public LogSettings GetDefaultStoreLogSettings(
-            bool useSeparatePageBlobStorage, 
-            long upperBoundOnAvailable, 
-            FasterTuningParameters tuningParameters)
+            return new FasterLogSettings
+            {
+                PageSizeBits = pageSizeBits,
+                SegmentSizeBits = segmentSizeBits,
+                 LogDevice = this.EventLogDevice,
+                LogCommitManager = this.UseLocalFiles
+                    ? null // TODO: fix this: new LocalLogCommitManager($"{this.LocalDirectoryPath}\\{this.PartitionFolderName}\\{CommitBlobName}")
+                    : (ILogCommitManager)this,
```

Comment:
can you please elaborate on this TODO comment? What needs fixing exactly?

Reply:
right, this was just a copy-paste (so it has nothing to do with this PR). The TODO here refers to the fact that at some point Netherite supported using the file system (instead of Azure Storage) to store task hubs. But it is not an official feature and commented out at the moment. There is some scenario in which we bring this back on K8s, therefore I did not want to just delete it outright.

Reply:
ah gotcha. That makes sense.
Might be helpful just to add a blurb to that comment that this local-files support is for K8s but not advertised today.